web/main.ts

```diff
@@ -45,6 +48,24 @@ function saveBoundsSoon() {
   }, 1000);
 }
 
+function loadFrontend() {
+  numberRetries--;
+  if (electronStore.get('development').embedded || numberRetries < 0) {
+    if(numberRetries < 0) {
+      const alertOptions: MessageBoxOptions = {
+        type: 'warning',
+        buttons: ['OK'],
+        message: `Reverted to Embedded Frontend because the Frontend proxy Url is not reachable.`,
+        detail: 'Make sure the front end service is running on Frontend proxy Url.',
```

Comment:
`Please ensure the frontend service is running at $<insert URL>.`

Reply:
✅